frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -244,6 +276,17 @@ export class WorkspaceAdvanced extends React.Component {
     })
   }
 
+  loadSubscriptionsRequests = async () => {
+    const { props, state } = this
+
+    const fetchSubscriptionsRequests = await handleFetchResult(await getSubscriptionsRequests(state.config.apiUrl, state.content.workspace_id))
+
+    switch (fetchSubscriptionsRequests.apiResponse.status) {
+      case 200: this.setState({ subscriptionsRequests: fetchSubscriptionsRequests.body.reverse() }); break
+      default: this.sendGlobalFlashMessage(props.t('Error while loading space requests', 'warning'))
```

Comment:
Are we clear that we always use "request" and never "subscription" everywhere in the UI by the way?

Reply:
Yes, "requests to join the space".